src/components/auth/ProtectedRoutes.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-console */
+import React from 'react';
+import { Outlet } from 'react-router';
+import Login from './Login';
+
+const useAuth = (obj) => obj && obj?.authenticated;
+
+function ProtectedRoutes(props) {
+  const obj = props;
+  const { user } = obj;
+  const isAuth = useAuth(user);
+  console.log(isAuth);
```

Comment:
- [ ] According to the javascript [requirements](https://github.com/microverseinc/curriculum-final-capstone/blob/main/projects/business_requirements.md#project-requirements):

![image](https://user-images.githubusercontent.com/57421823/212441356-e458334b-ab31-4916-9d2c-10d15b76e339.png)

Therefore, kindly remove this console.log debugging code. 👍

Reply:
all console.logs removed